e2e/rke2-capl-cluster/assert-capi-resources.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: capi-controller-manager
+  namespace: capi-system
+status:
+  availableReplicas: 1
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: capl-controller-manager
+  namespace: capl-system
+status:
+  availableReplicas: 1
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rke2-bootstrap-controller-manager
+  namespace: rke2-bootstrap-system
+status:
+  availableReplicas: 1
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rke2-control-plane-controller-manager
+  namespace: rke2-control-plane-system
+status:
+  availableReplicas: 1
+# ---
```

Comment:
were these commented out on purpose?

Reply:
Yeah I was trying to test out if the test would without checking on CRDs. I was seeing a weird error thats why